src/bindgen/ir/ty.rs

```diff
@@ -433,6 +433,16 @@ impl Type {
                 is_nullable: false,
                 is_ref: false,
             }),
+            "Box"
+                if config.language == Language::C && config.function.swift_name_macro.is_none() =>
```

Comment:
I think the swift_name_macro check is not correct... Can you elaborate on why you added it?

Reply:
I wasn't sure it was correct. I had no idea what `swift_name_macro` is used for!